configure.ac

```diff
@@ -207,15 +207,13 @@ AS_IF([test x$enable_asm != xno], [
         ], [
             AS_CASE([$host],
                 [*darwin*], [
-                    ASFLAGS="$ASFLAGS -f macho$BITTYPE -DPREFIX -DHAVE_ALIGNED_STACK=1" ],
+                    ASFLAGS="$ASFLAGS -f macho$BITTYPE -DPREFIX" ],
                 [*linux*|*dragonfly*|*bsd*|*solaris*|*haiku*], [
-                    ASFLAGS="$ASFLAGS -f elf$BITTYPE -DHAVE_ALIGNED_STACK=1" ],
+                    ASFLAGS="$ASFLAGS -f elf$BITTYPE" ],
```

Comment:
Won't this change stack alignment on 32-bit MacOS, Linux, Solaris and Haiku? Afaict x86inc defaults to what would previously have been `HAVE_ALIGNED_STACK=0` on i386-systems.
 *(Dragonfly and other BSDs shouldn't have 16Byte alignment on 32-bit anyway as demonstrated by #451 and fixed as part of #452)*